.gitignore

```diff
@@ -10,3 +10,4 @@
 /public/system
 /tags
 /tmp/*
+*.rdb
```

Comment:
Interesting. And this was being dumped in the app directory? I wonder why that started showing up on this branch. No harm in adding it, I suppose.